.github/workflows/main.yml

```diff
@@ -0,0 +1,88 @@
+name: Docker build and push
+
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+env:
+  # TEST_TARGET: Name of the testing target in the Dockerfile
+  TEST_TARGET: testing
+
+  # DO_TEST - true to build and run unit tests, false to skip the tests
+  DO_TEST: false
```

Comment:
I don't see this ever "going true" so maybe we remove it and the associated build steps